### PR TITLE
Display Job name in messages

### PR DESCRIPTION
**Pull request by bamandel**
There was no obvious place to insert this new data into the notification messages. In the future, we will want to add a step/object/processor that can allow easier modification of message content so we can add features without passing objects through the entire process.

### CODE REVIEW

File: api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/ProjectMessageConverter.java [hunk 1]
```diff
@@ -45,6 +45,11 @@ public ProjectMessageConverter(ChannelMessageFormatter formatter) {
             optionalProjectVersionString = Optional.empty();
         }
 
+        String nonBreakingSpace = messageFormatter.getNonBreakingSpace();
+
+        chunkedStringBuilder.append(String.format("Job%sname:%s%s", nonBreakingSpace, nonBreakingSpace, jobName));
+        chunkedStringBuilder.append(messageFormatter.getLineSeparator());
+
```
Comment (bamandel) [inline api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/ProjectMessageConverter.java:52]: Main change

File: api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/convert/ProjectIssueModelConverter.java [hunk 2]
```diff
@@ -58,11 +58,15 @@ public ProjectIssueModelConverter(IssueTrackerMessageFormatter formatter) {
...
-    public IssueCreationModel toIssueCreationModel(ProjectIssueModel projectIssueModel) {
+    public IssueCreationModel toIssueCreationModel(ProjectIssueModel projectIssueModel, String jobName) {
         String title = createTruncatedTitle(projectIssueModel);
 
         ChunkedStringBuilder descriptionBuilder = new ChunkedStringBuilder(formatter.getMaxDescriptionLength());
 
+        String nonBreakingSpace = formatter.getNonBreakingSpace();
+        descriptionBuilder.append(String.format("Job%sname:%s%s", nonBreakingSpace, nonBreakingSpace, jobName));
+        descriptionBuilder.append(formatter.getLineSeparator());
+
```
Comment (bamandel) [inline api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/convert/ProjectIssueModelConverter.java:69]: Main change

File: api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java [hunk 3]
```diff
@@ -23,9 +23,10 @@ public SimpleMessageConverter(ChannelMessageFormatter messageFormatter) {
     }
 
     @Override
-    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage) {
+    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage, String jobName) {
         ChunkedStringBuilder chunkedStringBuilder = new ChunkedStringBuilder(messageFormatter.getMaxMessageLength());
 
+        appendSection(chunkedStringBuilder, String.format("Job name: %s", jobName));
```
Comment (bamandel) [inline api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java:29]: Main change

File: api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java [hunk 4]
```diff
@@ -23,9 +23,11 @@ public SimpleMessageConverter(ChannelMessageFormatter messageFormatter) {
     }
 
     @Override
-    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage) {
+    public List<String> convertToFormattedMessageChunks(SimpleMessage simpleMessage, String jobName) {
         ChunkedStringBuilder chunkedStringBuilder = new ChunkedStringBuilder(messageFormatter.getMaxMessageLength());
 
+        String nonBreakingSpace = messageFormatter.getNonBreakingSpace();
+        appendSection(chunkedStringBuilder, String.format("Job%sname:%s%s", nonBreakingSpace, nonBreakingSpace, jobName));
```
Comment (bamandel) [inline api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java:30]: Main change

File: api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/convert/ProjectIssueModelConverter.java [hunk 5]
```diff
@@ -58,11 +58,15 @@ public ProjectIssueModelConverter(IssueTrackerMessageFormatter formatter) {
...
     }
 
-    public IssueCreationModel toIssueCreationModel(ProjectIssueModel projectIssueModel) {
+    public IssueCreationModel toIssueCreationModel(ProjectIssueModel projectIssueModel, String jobName) {
         String title = createTruncatedTitle(projectIssueModel);
 
         ChunkedStringBuilder descriptionBuilder = new ChunkedStringBuilder(formatter.getMaxDescriptionLength());
 
+        String nonBreakingSpace = formatter.getNonBreakingSpace();
+        descriptionBuilder.append(String.format("Job%sname:%s%s", nonBreakingSpace, nonBreakingSpace, jobName));
```
Comment (gkillough) [inline api-channel-issue-tracker/src/main/java/com/synopsys/integration/alert/api/channel/issue/convert/ProjectIssueModelConverter.java:67]: Do we want to make it bold or anything?

File: api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java (hunk 4 above)
Comment (gkillough) [reply api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java:30]: Bold?

**Review by gkillough — APPROVED** [review]
I think in the long run we will probably want to pass this through wrapped in a single object along with the channel messages, but for now this works.

File: api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java (hunk 4 above)
Comment (bamandel) [reply api-channel/src/main/java/com/synopsys/integration/alert/api/channel/convert/SimpleMessageConverter.java:30]: I'll go through the classes and bold the entire statement

**Pull request merged**